specs/src/specs/resource_pricing.md

```diff
@@ -168,12 +168,14 @@ entire fee is then deducted from the transaction no matter what.
 
 ## Fee market
 
-By default, Celestia consensus nodes will use mempools that prioritize fees,
-however mempool usage cannot be enforced at the protocol level. There is
-currently no enforced minimum fee, this value is set by each consensus node in
-their `app.toml`. Transactions that do not exceed that given gas price will not
-be able to enter that node's mempool, and thus they will also not be
-gossiped by that node.
+By default, Celestia's consensus nodes prioritize transactions in their mempools
+based on fees. In v1, there was no minimum fee enforced, allowing each 
```

Comment:
[not blocking] gas price seems more accurate. Tx A can have more fees than Tx B but whichever tx has a higher gas price will be prioritized higher.

```suggestion
By default, Celestia's consensus nodes prioritize transactions in their mempools
based on gas price. In v1, there was no minimum gas price enforced, allowing each 
```